platform-api/che-core-api-runner/src/test/java/org/eclipse/che/api/runner/RunnerServiceTest.java

```diff
@@ -283,29 +294,78 @@ public void testGetRunningProcesses() throws Exception {
 
 
         // we should have all the processes without specifying anything
-        processes = service.getRunningProcesses(workspaceCodenvy, projectA);
+        processes = service.getRunningProcesses(workspaceCodenvy, projectA, null);
         // list is 1 as we only have one matching workspace + project
         assertEquals(processes.size(), 1);
 
 
         // we should have all the processes without specifying anything
-        processes = service.getRunningProcesses(null, projectA);
+        processes = service.getRunningProcesses(null, projectA, null);
         // list is 2 as we have two workspaces with this project name
         assertEquals(processes.size(), 2);
 
         // no workspace with that name
-        processes = service.getRunningProcesses("dummy", null);
+        processes = service.getRunningProcesses("dummy", null, null);
         assertEquals(processes.size(), 0);
 
         // no project with that name
-        processes = service.getRunningProcesses(null, "project");
+        processes = service.getRunningProcesses(null, "project", null);
         assertEquals(processes.size(), 0);
 
         // no project with that workspace + name
-        processes = service.getRunningProcesses("dummy", "project");
+        processes = service.getRunningProcesses("dummy", "project", null);
         assertEquals(processes.size(), 0);
     }
 
+    /**
```

Comment:
I think javadocs is redundant in tests. Except private method and specific cases